pkg/cassandra/session.go

```
@@ -212,6 +213,23 @@ func toString(val interface{}) (string, error) {
 	return str, nil
 }
 
+// describe renders a scanned value for an error message. Timestamps get an
+// unambiguous layout, everything else falls back to its own formatting.
+func describe(val interface{}) string {
+	switch v := val.(type) {
+	case nil:
+		return "<null>"
+	case fmt.Stringer:
+		return v.String()
+	case time.Time:
```

golangci-lint (2.12.2)

[error] SA4020: unreachable case clause: fmt.Stringer will always match before time.Time

(staticcheck)